tests/test_ess_csc.py

```diff
@@ -47,9 +48,11 @@
 # Too long wait time for timeout tests (second).
 TOO_LONG_WAIT_TIME = 12
 
+PASCALS_PER_MILLIBAR = 100
```

Comment:
Maybe import the constant defined in the `lsst.ts.ess.csc` package so it is unambiguously defined?